test/TestProjects/Lro-Basic-Cadl/Generated/LroBasicCadlClient.cs

```diff
@@ -195,7 +195,7 @@ public virtual async Task<Operation<BinaryData>> UpdateProjectAsync(WaitUntil wa
             try
             {
                 using HttpMessage message = CreateUpdateProjectRequest(id, content, context);
-                return await ProtocolOperationHelpers.ProcessMessageAsync(_pipeline, message, ClientDiagnostics, "LroBasicCadlClient.UpdateProject", OperationFinalStateVia.Location, context, waitUntil).ConfigureAwait(false);
+                return await ProtocolOperationHelpers.ProcessMessageAsync(_pipeline, message, ClientDiagnostics, "LroBasicCadlClient.UpdateProject", OperationFinalStateVia.OperationLocation, context, waitUntil).ConfigureAwait(false);
```

Comment:
I think you are adding a new feature so why it would impact the existing code?

Reply:
Our polling implementation is adaptive. It actually doesn't require FinalStateVia to work properly. By the way, this code is out-of-date. There is no change in latest code.